service/project.go

```diff
@@ -40,39 +40,8 @@ func (uis *UIServer) filterViewableProjects(u *user.DBUser) ([]model.ProjectRef,
 }
 
 func (uis *UIServer) projectsPage(w http.ResponseWriter, r *http.Request) {
-	dbUser := MustHaveUser(r)
-
-	flags, err := evergreen.GetServiceFlags()
-	if err != nil {
-		gimlet.WriteResponse(w, gimlet.MakeJSONInternalErrorResponder(errors.Wrap(err, "retrieving admin settings")))
-	}
-	if flags.LegacyUIProjectPageDisabled {
-		newUIProjectsLink := fmt.Sprintf("%s/projects", uis.Settings.Ui.UIv2Url)
-		http.Redirect(w, r, newUIProjectsLink, http.StatusPermanentRedirect)
-	}
-
-	allProjects, err := uis.filterViewableProjects(dbUser)
-	if err != nil {
-		uis.LoggedError(w, r, http.StatusInternalServerError, err)
-		return
-	}
-
-	spruceLink := fmt.Sprintf("%s/projects", uis.Settings.Ui.UIv2Url)
-	newUILink := ""
-	if len(uis.Settings.Ui.UIv2Url) > 0 {
-		newUILink = spruceLink
-	}
-
-	slackAppName := uis.Settings.Slack.Name
-
-	data := struct {
-		AllProjects []model.ProjectRef
-		ViewData
-		NewUILink    string
-		SlackAppName string
-	}{allProjects, uis.GetCommonViewData(w, r, true, true), newUILink, slackAppName}
-
-	uis.render.WriteResponse(w, http.StatusOK, data, "base", "projects.html", "base_angular.html", "menu.html")
+	newUIProjectsLink := fmt.Sprintf("%s/projects", uis.Settings.Ui.UIv2Url)
+	http.Redirect(w, r, newUIProjectsLink, http.StatusPermanentRedirect)
 }
 
 func (uis *UIServer) projectPage(w http.ResponseWriter, r *http.Request) {
```

Comment:
Can we remove this too? And modifyProject?

Reply:
`projectPage` returns a JSON response of the project which is why I didn't include it as the removal, but I'm fine to get rid of it. Done.